staubli_rx160_gazebo/urdf/rx160.xacro

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" ?>
+<robot name="staubli_rx160_gazebo" xmlns:xacro="http://ros.org/wiki/xacro">
```

Comment:
Oh, sorry! That makes sense.
In the diff it appeared as highlighted lines 1 and 2, not the comment. :-S
Yes, we can remove that...